### DISCUSSION TITLE
Custom heights and widths are incompatible with class evenly in both flex and grid

### DISCUSSION THREAD

**Discussion by LeTruongThinh2002** [tailwindlabs/tailwindcss#13056] [Help]
last version

For example, I use w-[300px] h-[300px] or some size class to create a square shape for the 2 child divs and for the parent div outside the grid grid-cols-2 gap-4 content-evenly then content-evenly or justify-evenly or around or stretch will not work to balance the distance between 2 child divs, left and right inside the parent div

**Comment by wongjn** [comment]
Please consider providing a [Tailwind Play](https://play.tailwindcss.com) that exhibits the undesired behavior.
